src/Grid/LocalDataSource.ts

```diff
@@ -0,0 +1,332 @@
+import Axios from 'axios';
+import * as _ from 'lodash';
+import * as moment from 'moment';
+import * as Rx from 'rx';
+import { ColumnModel } from '.';
+import {
+  AggregateFunctions,
+  ColumnDataType,
+  ColumnSortDirection,
+  CompareOperators,
+} from './Column';
+import GridRequest from './GridRequest';
+import GridResponse from './GridResponse';
+import GridDataResponse from './utils/GridDataResponse';
+
+export default class LocalDataSource implements IDataSource {
+
+  public static defaultColumnValues = {
```

Comment:
Same here... this is duplicated on RemoteDataSource.

Reply:
Default columns values should be a static method at Column Model

Reply:
You need to remove this defaultColumnValues...